site/docs/releases.md

```diff
@@ -51,44 +51,60 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
+    * invoke stored procedures using CALL
+* **Flink** now supports streaming reads, CDC writes (experimental), and filter pushdown
+* **AWS module** is added to support better integration with AWS, with AWS Glue catalog support and dedicated S3 FileIO implementation
+* **Nessie module** is added to support integration with project Nessie
 
 Important bug fixes:
 
-* [\#1706](https://github.com/apache/iceberg/pull/1706) fixes non-vectorized ORC reads in Spark that incorrectly skipped rows
-* [\#1536](https://github.com/apache/iceberg/pull/1536) fixes ORC conversion of `notIn` and `notEqual` to match null values
-* [\#1722](https://github.com/apache/iceberg/pull/1722) fixes `Expressions.notNull` returning an `isNull` predicate; API only, method was not used by processing engines
-* [\#1736](https://github.com/apache/iceberg/pull/1736) fixes `IllegalArgumentException` in vectorized Spark reads with negative decimal values
-* [\#1666](https://github.com/apache/iceberg/pull/1666) fixes file lengths returned by the ORC writer, using compressed size rather than uncompressed size
-* [\#1674](https://github.com/apache/iceberg/pull/1674) removes catalog expiration in HiveCatalogs
-* [\#1545](https://github.com/apache/iceberg/pull/1545) automatically refreshes tables in Spark when not caching table instances
+* [\#2091](https://github.com/apache/iceberg/pull/2091) fixes Parquet vectorized reads when column types are promoted
+* [\#1991](https://github.com/apache/iceberg/pull/1991) fixes Avro schema conversions to preserve field docs
+* [\#1981](https://github.com/apache/iceberg/pull/1981) fixes date and timestamp transforms
+* [\#1962](https://github.com/apache/iceberg/pull/1962) fixes Parquet vectorized position reader
+* [\#1811](https://github.com/apache/iceberg/pull/1811) makes refreshing Spark cache optional
+* [\#1798](https://github.com/apache/iceberg/pull/1798) fixes read failure when encountering duplicate entries of data files
+* [\#1785](https://github.com/apache/iceberg/pull/1785) fixes invalidation of metadata tables in CachingCatalog
+* [\#1784](https://github.com/apache/iceberg/pull/1784) fixes resolving of SparkSession table's metadata tables
 
 Other notable changes:
 
-* The `iceberg-hive` module has been renamed to `iceberg-hive-metastore` to avoid confusion
-* Spark 3 is based on 3.0.1 that includes the fix for [SPARK-32168](https://issues.apache.org/jira/browse/SPARK-32168)
-* Hadoop tables will recover from version hint corruption
-* Tables can be configured with a required sort order
-* Data file locations can be customized with a dynamically loaded `LocationProvider`
-* ORC file imports can apply a name mapping for stats
+* NaN counter is added to format v2 metrics
+* Shared catalog properties are added in core library to standardize catalog level configurations
+* Spark and Flink now supports dynamically loading customized `Catalog` and `FileIO` implementations
+* Spark now supports loading tables with file paths via HadoopTables
```

Comment:
This can be removed. It is only to support the existing behavior in `IcebergSource` now that the source supports multi-catalog table references.